keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/resource/ImageHandler.kt

```diff
@@ -147,6 +149,8 @@ class ImageHandler(
         )
       )
     )
+    // TODO: check this
+    publisher.publishEvent(TaskCreatedEvent(TaskRecord(taskId = taskRef.taskId, taskName = description, resourceId = resource.id.toString())))
```

Comment:
Isn't this redundant with the event published in `TaskLauncher`?

Reply:
Will refactor this to use task launcher instead.